### ISSUE TITLE
fixed v0.5.5? django-b2 doesn't work when running on Windows

### ISSUE THREAD

**Issue by weaversam8** [pyutil/django-b2#2]
* django-b2 version: 0.5.0
* Django version: 3.0.4
* Python version: 3.7.3
* Operating System: Windows 10

### Description

Uploading a file on Windows doesn't work properly, because backslashes are not allowed in filenames. When uploading a file, this error appears:

![image](https://user-images.githubusercontent.com/2546219/82743589-5ef57f80-9d3b-11ea-921a-e2f3faba677c.png)

**Comment by weaversam8** [comment]
I monkey patched this by adding the following two lines into the `_save(...)` method in `storage.py`:

```python
def _save(self, name, f, max_length=None):
    if "\\" in name:
        name = name.replace("\\", "/")
    # ...
```

Not sure if this is the best solution, but it's a workaround for now.

**Comment by pyutil** [comment]
Hi, thank you very much.
I have moved your code to a little different place(s) to affect upload, download, deleting, listing,..
Not sure if everything works well.
Could you try django-b2 v 0.5.4 in Windows, at least for your scenario (upload)?

Note: In addition I made a change for Linux abs path which is now for b2 auto-converted to relative path (because on the b2 side it should be always relative ie. relative to the bucket).

**Comment by weaversam8** [comment]
I can confirm this is fixed in `v0.5.5`! Thanks!
